src-srv/utils/transformations/newsdoc/core/text.ts

```diff
@@ -26,9 +26,10 @@ const replace = (type: string, translateList: Record<string, string>): string =>
 }
 
 const createDocument = (): Document => {
+  // TODO: does this work, we'll find outj
```

Comment:
It seems like it's working? Should the comment be removed?

Reply:
Next attempt/PR is to revert and save. Then we'll find out